mattermost.go

```diff
@@ -71,7 +71,7 @@ var flagRunCmds bool
 func doLoadConfig(filename string) (err string) {
 	defer func() {
 		if r := recover(); r != nil {
-			err = r.(string)
+			err = fmt.Sprintf("Error loding config")
```

Comment:
Can you make this `fmt.Sprintf("Error loding config, err=%v", r)`? That should work to print out an error without having to type convert manually